conf/local.config

```diff
@@ -0,0 +1,5 @@
+process {
+    withLabel:dcqc {
+        container   = 'dcqc'
```

Comment:
Does this assume that you are building `py-dcqc` and naming the container `dcqc`?

Reply:
It's the default container image name when running `src/docker/build.sh` in the py-dcqc repo. I added a comment to clarify. 

https://github.com/Sage-Bionetworks-Workflows/py-dcqc/blob/main/src/docker/build.sh